docs/using-the-jdbc-driver/using-plugins/UsingTheSimpleReadWriteSplittingPlugin.md

```diff
@@ -0,0 +1,78 @@
+
+## Using the Simple Read/Write Splitting Plugin with RDS Proxy
+
+RDS Proxy provides connection pooling and management that significantly improves application scalability by reducing database connection overhead and enabling thousands of concurrent connections through
+connection multiplexing. Connecting exclusively through the proxy endpoint ensures consistent connection management, automatic failover handling, and centralized monitoring, while protecting the underlying database from connection exhaustion
+and providing a stable abstraction layer that remains consistent even when database topology changes. By providing the read/write endpoint and a read-only endpoint to the simple read/write splitting plugin, the AWS JDBC Driver will connect using 
+these endpoints any time setReadOnly is called. 
+
+To take full advantage of the benefits of RDS Proxy it is recommended to only connect through RDS Proxy endpoints. See [Using the AWS JDBC Driver with RDS Proxy](./../../../README.md#rds-proxy) for limitations.
+
+## Using the Simple Read/Write Splitting Plugin against non-Aurora clusters
+
+The simple read/write splitting plugin can be used to switch between any two endpoints. If the endpoints do not direct to an Aurora database, ensure the property `verifyNewSrwConnections` is set to `false`. See [Limitations of verifyNewSrwConnections](UsingTheSimpleReadWriteSplittingPlugin.md#non-aurora-clusters) for details.
+
+## Limitations
+
+### General plugin limitations
+
+When a Statement or ResultSet is created, it is internally bound to the database connection established at that moment. There is no standard JDBC functionality to change the internal connection used by Statement or ResultSet objects. Consequently, even if the read/write plugin switches the internal connection, any Statements/ResultSets created before this will continue using the old database connection. This bypasses the desired functionality provided by the plugin. To prevent these scenarios, an exception will be thrown if your code uses any Statements/ResultSets created before a change in internal connection. To solve this problem, please ensure you create new Statement/ResultSet objects after switching between the writer/reader.
+
+Verify plugin compatibility within your driver configuration using the [compatibility guide](../Compatibility.md).
+
+### Failover
+
+Immediately following a failover event, due to DNS caching, an Aurora cluster endpoint may connect to the previous writer, and the read-only endpoint may connect to the new primary instance. To avoid stale DNS connections, as well as errors/warnings from the verification logic we recommend including the [Aurora Initial Connection Strategy Plugin](./UsingTheAuroraInitialConnectionStrategyPlugin.md) in your configuration.
```

Comment:
I think that using `initialConnection` plugin ruins `srw` plugin. I don't think they're compatible.